connectors/java/databasetable/src/main/java/org/identityconnectors/databasetable/DatabaseTableConnector.java

```diff
@@ -1031,8 +1039,13 @@ private Set<AttributeInfo> buildAttributeInfoSet(ResultSet rset) throws SQLExcep
             log.ok("Name of the parameter being evaluated : {0}", name);
 
             final AttributeInfoBuilder attrBld = new AttributeInfoBuilder();
-            final int columnType = meta.getColumnType(i);
-            final String columnTypeName = meta.getColumnTypeName(i);
+            Integer columnType = meta.getColumnType(i);
+            String columnTypeName = meta.getColumnTypeName(i);
+
+            if(meta.getColumnType(i) == 5) {
```

Comment:
Type interpretation is handled in a separate project, the DB table connector has a dependency on. Please do not use this code snipped in this place and rather port this part of the contribution to the  "https://github.com/Evolveum/connector-legacy-libs/tree/master/dbcommon" repository. You can use parts of the commit "https://github.com/Evolveum/connector-legacy-libs/commit/6a80de70793b11ffe952632ca8f69ab46b1e26e2" as a reference fo the modifications needed for this.